src/components/Root/Root.js

```diff
@@ -168,15 +178,16 @@ class Root extends Component {
                   onError={config?.suppressIntlErrors ? () => {} : undefined}
                   defaultRichTextElements={this.defaultRichTextElements}
                 >
-                  <CalloutContext.Provider value={this.callout.current}>
+                  <CalloutContext.Provider value={this.state.callout}>
```

Comment:
isn't it a huge change, that previously `callout.current` were provided, but now it's `callout`?

Reply:
nope, it's still the same ref, the way of how it's created

previously `this.callout.current` was null by default, now `this.state.callout` is null
`this.callout.current` had a ref link, now it's `this.callout.current` 

so nothing is changed for consumers

but it would be nice to pass `this.callout` instead, in this case all consumers should be updated

Reply:
oh, right. thanks!